### ISSUE TITLE
AUR build fails

### ISSUE THREAD

**Issue by Marteon** [sardemff7/eventd#41]
Hello, I'd love to use eventd but sadly the build fails for me.
When trying to install from [AUR](https://aur.archlinux.org/packages/eventd/) `check()` fails.

[Here](https://pastebin.com/JeysYQtL) are the full logs.
Long story short that is the error: 
```
--- stderr ---
**
ERROR:../eventd-0.24.1/src/libnkutils/tests/token.c:972:_nk_token_list_tests_func: assertion failed (result == data->result): ("Thu M\303\244r  1 14:14:08 2018" == "Thu Mar  1 14:14:08 2018")
-------
```

**Comment by sardemff7** [comment]
Ow, locales…

Do you set `LC_ALL` specifically? If so, can you try unsetting it, or setting it to `C` (for the build)?
I guess I’ll just force `LC_ALL=C` from now on, instead of per-category.

I’ll push a fix (for the next release, and a workaround in the PKGBUILD) as soon as you can confirm it’ll work.

**Comment by Marteon** [comment]
Yes it works! I haven't set `LC_ALL` before but setting it to `C` fixed the problem!
Thank you for your quick reply and solution!

**Comment by sardemff7** [comment]
Pushed the workaround in the PKGBUILD and a fix as 17ce1be66c38d136f79cc5bd716e33c07f69ba45 (sardemff7/libnkutils@e3c2f9d52be26b561aa7b9ffc38ecf3763d36a02), to be in the next release.
